model/src/main/kotlin/spdx/SpdxExpression.kt

```diff
@@ -19,14 +19,21 @@
 
 package com.here.ort.model.spdx
 
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.databind.annotation.JsonSerialize
+import com.fasterxml.jackson.databind.ser.std.ToStringSerializer
+
 import org.antlr.v4.runtime.BaseErrorListener
 import org.antlr.v4.runtime.CharStreams
 import org.antlr.v4.runtime.CommonTokenStream
 import org.antlr.v4.runtime.RecognitionException
 import org.antlr.v4.runtime.Recognizer
 
+@JsonSerialize(using = ToStringSerializer::class)
 sealed class SpdxExpression {
     companion object {
+        @JsonCreator
+        @JvmStatic
```

Comment:
Nit: Should this line already be added in the previous commit? (Even if not technically needed there yet.)

Reply:
It's only added to enable Jackson to recognize the creator function, before it could still be used in Java with `SpdxExpression.Companion.parse()`, so it's more related to this commit I would say. 

We don't add `@JvmStatic` to our other static functions, but maybe we should add all those compatibility annotations to our code later in case someone wants to use the libraries from Java.

Reply:
> so it's more related to this commit I would say.

Agreed.

> but maybe we should add all those compatibility annotations to our code later

Again I agree, also about doing it later, eventually.